src/deploy/missions/contracts/z-token/z-token.ts

```diff
@@ -39,16 +39,16 @@ IZNSContracts
   >) {
     super(args);
 
-    if (this.config.mockMeowToken) {
-      this.contractName = znsNames.meowToken.contractMock;
+    if (this.config.mockZToken) {
```

Comment:
now here it is a boolean already, so you can check it like this without problems